lxd/container.go

```diff
@@ -227,11 +227,17 @@ func containerValidDeviceConfigKey(t, k string) bool {
 		}
 	case "proxy":
 		switch k {
-		case "listen":
```

Comment:
Can you sort those alphabetically (looks like you started to do that but then didn't do it all the way :))

Reply:
Ah, i switched `connect` and `gid` accidently.